### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ljtill**

### CODE REVIEW

**Review by jtracey93 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -22,10 +26,44 @@ For information on contributing to the module, visit the [Contributing Guide](ht
...
 File an issue via [GitHub Issues](https://github.com/azure/azops/issues/new/choose)  
 
-### Issues
+### Output
 
-![GitHub issues by-label](https://img.shields.io/github/issues/azure/azops/enhancement%20:bulb:?label=enhancement%20issues)
-![GitHub issues by-label](https://img.shields.io/github/issues/azure/azops/bug%20:ambulance:?label=bug%20issues)
+AzOps is rooted in the principle that everything in Azure is a resource and to operate at-scale, it should be managed declaratively to determine target goal state of the overall platform. This PowerShell module provides the ability to deploy Resource Templates & Bicep files at all Azure [scope](https://docs.microsoft.com/azure/role-based-access-control/scope-overview) levels. To provide this functionility the multiple scopes within Azure Resource Manager are represented (example below) within Git. Using directories and files, templates can be deployed (Push) at various scopes whilst also exporting composite templates from ARM and placing them within the repository.
+
+```bash
```
Comment (jtracey93) [inline README.md:33]: Do we change this to a screenshot for ease to consume?

File: README.md [hunk 2]
```diff
@@ -22,10 +26,44 @@ For information on contributing to the module, visit the [Contributing Guide](ht
 For information on migrating to the new version, visit the [Migration Guide](https://github.com/azure/azops/wiki/migration)  
 File an issue via [GitHub Issues](https://github.com/azure/azops/issues/new/choose)  
 
-### Issues
+### Output
 
-![GitHub issues by-label](https://img.shields.io/github/issues/azure/azops/enhancement%20:bulb:?label=enhancement%20issues)
-![GitHub issues by-label](https://img.shields.io/github/issues/azure/azops/bug%20:ambulance:?label=bug%20issues)
+AzOps is rooted in the principle that everything in Azure is a resource and to operate at-scale, it should be managed declaratively to determine target goal state of the overall platform. This PowerShell module provides the ability to deploy Resource Templates & Bicep files at all Azure [scope](https://docs.microsoft.com/azure/role-based-access-control/scope-overview) levels. To provide this functionility the multiple scopes within Azure Resource Manager are represented (example below) within Git. Using directories and files, templates can be deployed (Push) at various scopes whilst also exporting composite templates from ARM and placing them within the repository.
```
Comment (jtracey93) [inline README.md:31]:
```suggestion
AzOps is rooted in the principle that everything in Azure is a resource and to operate at-scale, it should be managed declaratively to determine target goal state of the overall platform. This PowerShell module provides the ability to deploy Resource Templates & Bicep files at all Azure [scope](https://docs.microsoft.com/azure/role-based-access-control/scope-overview) levels. To provide this functionility the multiple scopes within Azure Resource Manager are represented (example below) within Git. Using directories and files, templates can be deployed (Push) at various scopes whilst also exporting composite templates from ARM and placing them within the repository (Pull).
```

**Review by jtracey93 — COMMENTED** [review]
Just a couple of things for thought?

File: README.md (hunk 1 above)
Comment (ljtill) [reply README.md:33]: Let's stick with the tree representation for now and I'll re-deploy the environment to get the new screenshots in the next few days.

**Review by jtracey93 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**